protocol/messenger_profile_showcase.go

```diff
@@ -287,6 +287,9 @@ func (m *Messenger) GetProfileShowcaseForSelfIdentity() (*protobuf.ProfileShowca
 }
 
 func (m *Messenger) BuildProfileShowcaseFromIdentity(state *ReceivedMessageState, message *protobuf.ProfileShowcase) error {
+	if message == nil {
```

Comment:
Could you please move this check [here](https://github.com/status-im/status-go/blob/develop/protocol/messenger_handler.go#L2930)? I did not noticed this PR an d i created another similar [PR]( https://github.com/status-im/status-go/pull/4298)